CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 1.0.0
```

Comment:
Any specific reason of bumping the pack version from `0.2.1` to `1.0.0`? I don't believe we're introducing any breaking changes in this PR.

Reply:
You're right, I've corrected the version.